From 33ac60cdc1bcc87a7eda7a7ce97b4fcce6416ff4 Mon Sep 17 00:00:00 2001
From: Alessandro Fontana <alessandro.fontana@pm.me>
Date: Fri, 9 Oct 2026 18:16:23 +0200
Subject: [PATCH 01/33] mainline/generic: Allow overriding suspend support

Assign TARGET_SUPPORTS_SUSPEND with ?= so that a device inheriting
device.mk can enable suspend.

Assisted-by: LLM
Change-Id: I3a48eeba1585c5f85721839eb1e0b5950c5612bc
---
 device.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/device.mk b/device.mk
index 9e91cf6..f6688ce 100644
--- a/device.mk
+++ b/device.mk
@@ -17,7 +17,7 @@ TARGET_GRAPHICS_COMPOSER_HAL := custom
 TARGET_HEALTH_HAL := custom
 TARGET_HOSTAPD_AND_WPA_SUPPLICANT_FORM := legacy
 TARGET_MESA_DO_NOT_SET_AS_DEFAULT := true
-TARGET_SUPPORTS_SUSPEND := false
+TARGET_SUPPORTS_SUSPEND ?= false
 TARGET_USES_TABLET_INPUT_AS_TOUCHSCREEN := true
 include device/mainline/common/optional/options.mk
 $(call inherit-product, device/mainline/common/mainline_common.mk)
-- 
2.47.3

